_integration_tests/test/notifications/cronjob_created_notification.js

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
'use strict' fehlt hier.

```suggestion
'use strict';

/* eslint-disable @typescript-eslint/no-var-requires */
```